arangod/RocksDBEngine/RocksDBKey.cpp

```diff
@@ -29,6 +29,8 @@
 #include "RocksDBEngine/RocksDBFormat.h"
 #include "RocksDBEngine/RocksDBTypes.h"
 
+#include <iostream>
```

Comment:
Needed?

Reply:
I think so, as the file is implementing `operator<<` to append to an `std::ostream`.
It already did so before, but the required headers were missing.